ui/component/core/src/index.ts

```diff
@@ -848,8 +735,8 @@ export class Manager implements EventProviderFactory {
         }
 
         let result: BasicLoginResult = {
-            username: this._config.credentials?.username ? this._config.credentials?.username : "",
-            password: this._config.credentials?.password ? this._config.credentials?.password : "",
+            username: this.config.credentials?.username ? this.config.credentials?.username : "",
+            password: this.config.credentials?.password ? this.config.credentials?.password : "",
```

Comment:
Is there a reason to use `config` instead of `_config` here?
I don't exactly know the difference between them...

Reply:
`this.config` is just using the getter which seems more appropriate but not that important